### ISSUE TITLE
Update benchmark notebook to fetch cloud instance type value

### ISSUE THREAD

**Issue by dchaley** [dchaley/deepcell-imaging#45]
The notebook has a placeholder for getting current machine type. Fix this!

**Comment by dchaley** [comment]
Blocked on #46. Can't run on cloud without model download 🤦

**Comment by lynnlangit** [comment]
add instructions on how to create the expected model location (and which file to upload) at `.\keras....` on the README page [here](https://github.com/dchaley/deepcell-imaging/blob/main/benchmarking/deepcell-e2e/README.md), so that the e-2-e notebook can be run for testing

**Comment by dchaley** [comment]
Separate issue– see also #46

**Comment by dchaley** [comment]
Renamed this issue for clarity

**Comment by dchaley** [comment]
Fixed by #65

**Comment by dchaley** [comment]
Example output:
```
"mesmer-sample-3-dev",4.19,"2023-12-05 06:11:27.137393+00:00","n1-standard-1","todo-gpu-type","todo-gpu-count","base",True,17.58,0.0,0.75,16.82,0.08,0.34,15.89,0.52,"0.12.9"
```

nb: `n1-standard-1`
